config/config.go

```diff
@@ -106,11 +106,22 @@ func handleIncludes(baseDir string, content []byte) (string, error) {
 			final = append(final, line)
 			continue
 		}
-		includedCfg, err := readConfigFile(filepath.Join(baseDir, m[1]))
+
+		files, err := filepath.Glob(filepath.Join(baseDir, m[1]))
 		if err != nil {
 			return "", err
 		}
-		final = append(final, includedCfg)
+		if len(files) == 0 {
+			return "", errors.New("config include missing file")
```

Comment:
I think we can provide more information in the error message:

```suggestion
			return "", fmt.Errorf("config include error: %s didn't match any files", m[1])
```